openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/index.tsx

```diff
@@ -69,126 +67,109 @@ export const WhatsNewModal: FunctionComponent<Props> = ({
+          {header}
+        </Typography.Text>
+      }
+      visible={visible}
+      width={1200}>
+      <div className="flex w-auto h-full">
+        <div
+          className="border-r-2 p-x-md p-y-md border-separate"
+          style={{ width: '14%' }}>
+          <div className="flex flex-col-reverse">
+            {WHATS_NEW.map((d) => (
+              <div className="flex items-center justify-end mb-2.5" key={d.id}>
+                <svg
+                  fill="none"
+                  height="1em"
+                  viewBox="0 0 13 6"
+                  width="1em"
+                  xmlns="http://www.w3.org/2000/svg">
+                  <path
+                    clipRule="evenodd"
+                    d={iconString}
+                    fill={activeData.id === d.id ? '#7147E8' : '#6B7280'}
+                    fillRule="evenodd"
+                  />
+                </svg>
+                <button
+                  className={classNames(
+                    'm-l-xss',
+                    activeData.id === d.id ? 'text-primary' : null
+                  )}
+                  onClick={() => setActiveData(d)}>
+                  {d.version}
+                </button>
+              </div>
+            ))}
           </div>
         </div>
-        <div
-          className="tw-modal-body tw-p-0 tw-overflow-hidden"
-          style={{ height: '720px' }}>
-          {/* body */}
-          <div className="tw-flex tw-w-auto tw-h-full">
-            <div
-              className="tw-border-r-2 tw-px-4 tw-py-4 tw-border-separate"
-              style={{ width: '14%' }}>
-              <div className="tw-flex tw-flex-col-reverse">
-                {WHATS_NEW.map((d) => (
-                  <div
-                    className="tw-flex tw-items-center tw-justify-end tw-mb-2.5"
-                    key={d.id}>
-                    <svg
-                      fill="none"
-                      height="1em"
-                      viewBox="0 0 13 6"
-                      width="1em"
-                      xmlns="http://www.w3.org/2000/svg">
-                      <path
-                        clipRule="evenodd"
-                        d={iconString}
-                        fill={activeData.id === d.id ? '#7147E8' : '#6B7280'}
-                        fillRule="evenodd"
-                      />
-                    </svg>
+        <div className="overflow-y-auto" style={{ width: '86%' }}>
+          <div className="p-t-md px-10 ">
+            <div className="flex justify-between items-center p-b-sm">
+              <div>
+                <p className="text-base font-medium">{activeData.version}</p>
+                <p className="text-grey-muted text-xs">
+                  {activeData.description}
+                </p>
+              </div>
+              <div>
+                <div
+                  className={classNames(
+                    'inline-flex border-1 border border-primary text-sm rounded-6 h-8 bg-white',
```

Comment:
Same here. Can we avoid using more CSS like this? I would suggest using antd components and giving styling in a single class if required. Please do let me know if you think otherwise.

Reply:
Sure.